lib/Client/AbstractClient.php

```diff
@@ -13,7 +13,7 @@
 abstract class AbstractClient
 {
     /**
-     * @var OptionsResolver
+     * @var OptionsResolver|null
```

Comment:
Is this really true?

Reply:
Theoretically it's correct since it's populated in the getter and you do a null check in it on line 46

Reply:
Yes, you are correct. 

Thank you